lib/traject/readers/folio_postgres_reader.rb

```diff
@@ -312,6 +345,9 @@ def sql_query(conditions, addl_from: nil)
           ON parentItem.holdingsRecordId = parentHolding.id
       LEFT JOIN sul_mod_inventory_storage.instance parentInstance
           ON parentHolding.instanceid = parentInstance.id
+      -- Requests relation
+      LEFT JOIN sul_mod_circulation_storage.request request
+          ON (request.jsonb ->> 'instanceId')::uuid = vi.id
```

Comment:
Does this retrieving every request for an instance? Can we further filter it by request status just to the open/waiting request?

Reply:
Yeah, I think we should probably filter the requests to just the one's we're interested in.